### PR TITLE
[Debug] Fixed error handling when an error is already handled when another error is already handled (5)

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

---

Please, don't ask how to reproduce it :)

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@lyrixx Can you rebase on 3.4?

**Comment by lyrixx** [comment]
@fabpot Oups, I made the fix against 3.4, but forgot to update the base branch. It's now fixed.

File: src/Symfony/Component/Debug/ExceptionHandler.php [hunk 1]
```diff
@@ -388,7 +388,11 @@ private function formatPath($path, $line)
 
             $link = strtr($fmt[0], ['%f' => $path, '%l' => $line]);
         } else {
-            $link = $fmt->format($path, $line);
+            try {
+                $link = $fmt->format($path, $line);
+            } catch (\Exception $e) {
+                return sprintf('<span class="block trace-file-path">in <span title="%s%3$s"><strong>%s</strong>%s</span></span>', $this->escapeHtml($path), $file, 0 < $line ? ' line '.$line : '');
```
Comment (lyrixx) [inline src/Symfony/Component/Debug/ExceptionHandler.php]: I could use a goto here to avoid code duplication and re-use the line 375... :D

**Review by nicolas-grekas — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**